TelCo.ColorCoder/ManualMap.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+  class ManualMap{
+    
+    public static void ToString(){
+      for(int pairNumber = 1; pairNumber < 26; pairNumber++){
+
+          ColorPair testPair1 = ColorToNumber.GetColorFromPairNumber(pairNumber);
+          Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
```

Comment:
If you see program.cs version of this exercise - the console.WriteLine() was only in main function().
Similarly it is good to separate the functionality from the console.writeline.
The class responsibility is to provide a "ToString()" method which will return string representation of the manual. The string can be either printed or done a Console.WriteLine

Is this comment clear?

Reply:
Yes, It is clear. I will separate the processing and display functionality. Thank You for the feedback!